source/incident-collaboration/launching-playbooks.rst

```diff
@@ -48,23 +46,23 @@ Tasks can be part of pre-configured task templates in playbooks and they can als
 
 .. image:: ../images/IC-ad-hoc-tasks.gif
 
-Changing commanders
-~~~~~~~~~~~~~~~~~~~
+Changing owners
+~~~~~~~~~~~~~~~
 
-To change commanders you can run the ``/incident commander @username`` slash command from the incident channel. You can also use the desktop or browser to:
+To change playbook owners you can run the ``/playbook owner @username`` slash command from the playbook's channel. You can also use the desktop or browser to:
 
-1. Select the **Incidents** icon in the channel header to open the right-hand sidebar.
+1. Select the **Playbooks** icon in the channel header to open the right-hand sidebar.
 2. Select the **Summary** tab.
-3. Within the **Summary** tab, select the current commander's name in the right-hand sidebar, then select the new commander.
-  * To change the commander to a user who is not in the channel, first add the user to the channel.
+3. Within the **Summary** tab, select the current owner's name in the right-hand sidebar, then select the new owner.
+  * To change the owner to a user who is not in the channel, first add the user to the channel.
 
 Providing a status update
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Incident status updates ensure that stakeholders remain informed about the progress toward incident resolution. To post a status update:
+Status updates ensure that stakeholders remain informed about the playbook run's progress. To post a status update:
 
-1. Select the **Incidents** icon in the channel header to open the right-hand sidebar.
-2. Find the incident you're looking for and select **Go to Incident Channel**.
+1. Select the **Playbook** icon in the channel header to open the right-hand sidebar.
+2. Find the playbook you're looking for and select **Go to Playbook**.
```

Comment:
The button reads `Go to channel`
![image](https://user-images.githubusercontent.com/3924815/125490774-8c484287-a45e-46df-bce9-c304fba372b5.png)
```suggestion
2. Find the playbook you're looking for and select **Go to channel**.
```